### ISSUE TITLE
406 VipsJpeg: Premature end of input file

### ISSUE THREAD

**Issue by WatchDogsDev** [cshum/imagor#124]
Hi, I recently upgraded the imagor to v1.0.0 and I saw this error in docker container log many times for different images:

```
{
    "level": "warn",
    "ts": 1660161069.1208205,
    "caller": "imagor/imagor.go:326",
    "msg": "process",
    "params": {
        "path": "fit-in/1280x720/filters:format(webp)/xxxx",
        "image": "xxxx",
        "hash": "xxxxx",
        "fit_in": true,
        "width": 1280,
        "height": 720,
        "filters": [
            {
                "name": "format",
                "args": "webp"
            }
        ]
    },
    "error": "imagor: 406 VipsJpeg: Premature end of input file"
}
```

**Comment by cshum** [comment]
Can you share a few images that you are seeing the issue? Which Loader/Storage are you using?

**Comment by WatchDogsDev** [comment]
I'm using the HTTP Loader.

I tried to reproduce the issue but I couldn't cause when I try to process the image which gave error in production, Everything is ok! 

I don't think that the problem comes from images.

**Comment by cshum** [comment]
Which imagor version are you using before upgrading to v1.0.0 ?

**Comment by WatchDogsDev** [comment]
> Which imagor version were you using before upgrading to v1.0.0 ?

I was using 0.9.13

I will also try to reproduce the issue in development environment. 

Thanks ;)

**Comment by cshum** [comment]
I made some tweaks to the master branch. Would you also try the master branch docker image?
```
ghcr.io/cshum/imagor:master
```
You may want to clear the image cache if you have pull that before.

**Comment by cshum** [comment]
This should be fixed in v1.1.3. The remaining is likely due to corrupted files remaining in storage, if storage is enabled. For that you may perform a cleanup based on images seen from error log.
